ballista/rust/core/src/execution_plans/shuffle_writer.rs

```diff
@@ -492,27 +492,9 @@ mod tests {
     use datafusion::physical_plan::expressions::Column;
     use datafusion::physical_plan::limit::GlobalLimitExec;
     use datafusion::physical_plan::memory::MemoryExec;
-    use std::borrow::Borrow;
+    use datafusion::field_util::StructArrayExt;
     use tempfile::TempDir;
 
-    pub trait StructArrayExt {
```

Comment:
moved to `datafusion::field_util::StructArrayExt` to share it with physical_plan/expressions/get_indexed_field